src/WebJobs.Script/Utility.cs

```diff
@@ -591,7 +590,7 @@ public static void ValidateBinding(BindingMetadata bindingMetadata)
         {
             if (bindingMetadata.Name == null || !BindingNameValidationRegex.IsMatch(bindingMetadata.Name))
             {
-                throw new ArgumentException($"The binding name {bindingMetadata.Name} is invalid. Please assign a valid name to the binding.");
+                throw new ArgumentException($"The binding name {bindingMetadata.Name} is invalid. Please assign a valid name to the binding. See https://aka.ms/azure-functions-binding-name-rules for more details.");
```

Comment:
Fwlink has already been created. Documentation will go live once the host release for this is complete.

Reply:
Do we have an issue tracking that? The link would need to be updated to point to the right anchor, right?

Reply:
https://github.com/Azure/azure-functions-host/issues/10781 - yes, this link will be updated to point to the right anchor as soon as we publish the docs officially. Working with @mattchenderson and @ggailey777 on this on the side.